content/de/cuba/modules/gui/src/com/haulmont/cuba/gui/messages_de.properties

```diff
@@ -110,6 +110,7 @@ com.haulmont.cuba.gui.app.core.entityinspector/exportZIP = Als ZIP exportieren
 com.haulmont.cuba.gui.app.core.entityinspector/general = Allgemein
 com.haulmont.cuba.gui.app.core.entityinspector/import = Import
 com.haulmont.cuba.gui.app.core.entityinspector/importFailed = Import fehlgeschlagen
+com.haulmont.cuba.gui.app.core.entityinspector/importFailedMessage = Möglicherweise, hat die Datei '%s' einen falschen Inhalt. %s
```

Comment:
i would remove the colon after "Möglicherweise". In english it says "Probably" --> "Wahrscheinlich", here the weaker form "Möglicherweise" which is more like "Possibly".

As the importFailedMessage normally is based on the content of the entity inspector file, i would go with the stronger translation.

What do you think of:
`Wahrscheinlich ist der Inhalt der Date '%s' inkorrekt. %s`